rpc/contract.go

```diff
@@ -39,6 +42,17 @@ func (h *Handler) StorageAt(address, key felt.Felt, id BlockID) (*felt.Felt, *js
 	}
 	defer h.callAndLogErr(stateCloser, "Error closing state reader in getStorageAt")
 
+	// This checks if the contract exists because if a key doesn't exist in contract storage,
+	// the returned value is always zero and error is nil.
+	_, err := stateReader.ContractNonce(&address)
+	if err != nil {
+		if errors.Is(err, db.ErrKeyNotFound) {
+			return nil, ErrContractNotFound
+		}
+		h.log.Errorw("Failed to get contract nonce", "err", err)
```

Comment:
As far as i remember logging an Error would cause Juno to shut down, is that the behaviour you want?